helm/dagster/schema/schema_tests/test_user_deployments.py

```diff
@@ -30,6 +31,16 @@ def full_helm_template() -> HelmTemplate:
     )
 
 
+@pytest.fixture(name="subchart_template")
+def subchart_helm_template() -> HelmTemplate:
+    return HelmTemplate(
+        helm_dir_path="helm/dagster/charts/dagster-user-deployments",
```

Comment:
So this is an interesting thing.

In this test suite, there's already two other fixtures: `helm_template` and `full_helm_template`.

I don't think either is capable of testing a standalone deployment of the user subchart, since they're both run (`helm_dir_path`) from the outer chart directory.

This fixture, being run from the subchart directory with no additional subchart of its own, better simulates a standalone deployment.